EIPS/eip-4886.md

```diff
@@ -0,0 +1,294 @@
+
+ Possibly even more damaging is the effect on user confidence across the whole ecosystem. Users become reluctant to interact with apps and smart contracts for fear of putting their assets at risk. They may also decide not to store assets in cold wallet addresses as they need to prove they own them on a regular basis. A pertinent example is the user trying to decide whether to 'vault' their NFT and lose access to a discord channel, or keep their NFT in another wallet, or even to connect their 'vault' to discord.
+
+ Ethereum is amazing at providing trustless proofs. The *only* time a user should need to interact using the wallet that holds an asset is if they intend to sell or transfer that asset. If a user merely wishes to prove ownership (to access a resource, get an airdrop, mint an NFT, or vote in a DAO), they should do this through a trustless proof stored on-chain.
+
+ Furthermore, users should be able to decide where new assets are delivered, rather than them being delivered to the wallet providing the interaction. This allows hot wallets to acquire assets sent directly to a cold wallet 'vault', possibly even the one they are representing in terms of asset ownership.
+
+ The aim of this EIP is to provide a convenient method to avoid this security concern and empower more people to feel confident leveraging the full scope of Ethereum functionality. Our vision is an Ethereum where users setup a new hardware wallet for assets they wish to hold long-term, then make one single contract interaction with that wallet: to nominate a hot wallet proxy. That user can always prove they own assets on that address, and they can specify it as a delivery address for new asset delivery.
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+### Definitions
+
+ * Delivery address: The address that assets will be delivered to for the current Proxy Record, i.e. a new NFT minted by the Proxy address, representing the Nominator address, should be delivered to the Delivery address.
+ * Nomination: Where a Nominator has nominated a Proxy address. Will only be active when the Proxy has accepted the nomination.
+ * Nominator address: The address that proposes a proxy relationship. This address nominates another address to act as it's proxy, representing it and  it's holdings in all interations.
+ * Proxy address: The address that will represent a Nominator on-chain.
+ * Proxy Record: An active proxy relationship encompassing a Nominator, Proxy and Delivery.
+ * Register: The main EPS contract, which holds details of both Nominations and Proxy Records.
+
+### EPS Specification 
+
+There are two main parts to the register - a nomination and a proxy record:
+
+    Contract / Dapp                        Register
+
+    Nominator: 0x1234..             Nominator: 0x1234..
```

Comment:
I think you should put these diagrams in a code block to ensure the monospacing is preserved.